packages/components/src/ui/data-table-filter/lib/debounce.ts

```
@@ -0,0 +1,138 @@
+  function trailingEdge(time: number): ReturnType<T> | undefined {
+    timeout = null
+    if (trailing && lastArgs) {
+      return invokeFunc(time)
+    }
+    lastArgs = null
+    lastThis = null
+    return result
+  }
+
+  function debounced(
+    this: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)